exporters/otlp/otlptrace/otlptracegrpc/exporter.go

```diff
@@ -18,27 +18,14 @@ import (
 	"context"
 
 	"go.opentelemetry.io/otel/exporters/otlp/otlptrace"
-	tracesdk "go.opentelemetry.io/otel/sdk/trace"
 )
 
-// NewExporter constructs a new Exporter and starts it.
-func NewExporter(ctx context.Context, opts ...Option) (*otlptrace.Exporter, error) {
-	return otlptrace.NewExporter(ctx, NewClient(opts...))
+// New constructs a new Exporter and starts it.
+func New(ctx context.Context, opts ...Option) (*otlptrace.Exporter, error) {
+	return otlptrace.New(ctx, NewClient(opts...))
 }
 
 // NewUnstartedExporter constructs a new Exporter and does not start it.
 func NewUnstartedExporter(opts ...Option) *otlptrace.Exporter {
```

Comment:
```suggestion
func NewUnstarted(opts ...Option) *otlptrace.Exporter {
```

Should this drop `Exporter` for consistency with the outgoing `otlp` exporter?